crates/client/db/src/lib.rs

```diff
@@ -66,33 +61,15 @@ pub(crate) async fn open_rocksdb(
 
     opts.set_env(&env);
 
-    let backup_hande = if let Some(backup_dir) = backup_dir {
-        let (restored_cb_sender, restored_cb_recv) = oneshot::channel();
-
-        let (sender, receiver) = mpsc::channel(1);
```

Comment:
what is the mpsc channel for?

Reply:
backup restore task, i moved elsewhere so that DeoxysBackend::open_for_testing is not async for convenience

when backups are enabled a thread is spawned that owns the rocksdb BackupEngine (it is not thread safe) and it receives backup requests using a mpsc channel
There is also another oneshot channel involved: when restoring the db at startup is enabled, we want to wait on the backupengine to finish before returning from `open()`

I will add docs about that